src/components/application_manager/src/application_impl.cc

```diff
@@ -228,7 +228,8 @@ void ApplicationImpl::set_voice_communication_supported(
 }
 
 bool ApplicationImpl::IsAudioApplication() const {
-  return is_media_ || is_voice_communication_application_ || is_navi_;
+  return is_media_ || is_voice_communication_application_ || is_navi_ ||
+         mobile_projection_enabled_;
```

Comment:
why it is or with mobile_projection_enabled_?
mobile_projection app can be media or non-media. For non-media mobile projection app, it is wrong.

Reply:
@yang1070 it was fixed in https://github.com/smartdevicelink/sdl_core/tree/feature/mobile_projection_2